config/500-controller.yaml

```diff
@@ -69,6 +69,11 @@ spec:
         - name: METRICS_DOMAIN
           value: knative.dev/net-istio
 
+        # Improve provisioning time of new or updated kservice in large clusters
+        # see https://github.com/knative/networking/pull/1001
+        - name: PROBE_MAX_RETRY_DELAY_SECONDS
+          value: "5"
```

Comment:
I would rather keep the default here and a note in docs or here as a comment about large clusters.

Reply:
I am okay to change to the default here. Before I start seaching, can you point me to a good place where to document this?

What I do not see is the higher traffic issue. In a cluster with KServices that are really used, no matter if it is small or large, the amount of probe requests relative to all requests will be low if not irrelevant.

Reply:
> the amount of probe requests relative to all requests
I depends on the traffic I guess. What I mean is that I want to avoid something observable from other users.